pkg/keyspace/tso_keyspace_group.go

```diff
@@ -1013,6 +1013,80 @@ func (m *GroupManager) FinishMergeKeyspaceByID(mergeTargetID uint32) error {
 	return nil
 }
 
+// MergeAllIntoDefaultKeyspaceGroup merges all other keyspace groups into the default keyspace group.
+func (m *GroupManager) MergeAllIntoDefaultKeyspaceGroup() error {
+	mergedGroupNum := 0
+	for i := 0; i < int(endpoint.UserKindCount); i++ {
+		userKind := endpoint.UserKind(i)
```

Comment:
We didn't consider the user kind in other methods right now.

Reply:
It's just used to iterate the whole `groups` map here, maybe we could simply keep it.